src/radical/entk/execman/base/task_manager.py

```diff
@@ -158,7 +158,7 @@ def _sync_with_master(self, obj, obj_type, channel, queue):
         reply_queue = '-'.join(list(reversed(qname)))
         reply_queue = sid + '-' + reply_queue
 
-        while True:
+        while False:
```

Comment:
This should also be commented out, if possible with the explanation you wrote in the PR comments.

Reply:
Thanks again, 9e148bc, 8ec40a5